diff --git a/node_modules/react-native-youtube-iframe/lib/commonjs/PlayerScripts.js b/node_modules/react-native-youtube-iframe/lib/commonjs/PlayerScripts.js
index 7867fe7..11e3631 100644
--- a/node_modules/react-native-youtube-iframe/lib/commonjs/PlayerScripts.js
+++ b/node_modules/react-native-youtube-iframe/lib/commonjs/PlayerScripts.js
@@ -181,6 +181,7 @@ const MAIN_SCRIPT = (videoId, playList, initialPlayerParams, allowWebViewZoom, c
             end: ${end},
             rel: ${rel_s},
             playsinline: 1,
+            autoplay: 1,
             loop: ${loop_s},
             color: ${color},
             start: ${start},
diff --git a/node_modules/react-native-youtube-iframe/src/PlayerScripts.js b/node_modules/react-native-youtube-iframe/src/PlayerScripts.js
index 547b51f..acb4493 100644
--- a/node_modules/react-native-youtube-iframe/src/PlayerScripts.js
+++ b/node_modules/react-native-youtube-iframe/src/PlayerScripts.js
@@ -201,6 +201,7 @@ export const MAIN_SCRIPT = (
             end: ${end},
             rel: ${rel_s},
             playsinline: 1,
+            autoplay: 1,
             loop: ${loop_s},
             color: ${color},
             start: ${start},
